### PR TITLE
feat(training): add the ability to switch list

### PR DESCRIPTION
**Pull request by dbl7**

### CODE REVIEW

File: src/app/training/pages/training-page/training-page.component.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
+@import "../../../../styles/variables.scss";
```
Comment (dgutoryan) [inline src/app/training/pages/training-page/training-page.component.scss:1]: What about `stylePreprocessorOptions` option?

File: src/app/training/components/list-switch/list-switch.component.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
+@import "../../../../styles/variables.scss";
```
Comment (dgutoryan) [inline src/app/training/components/list-switch/list-switch.component.scss:1]: What about `stylePreprocessorOptions` option?

File: src/app/training/components/list-switch/list-switch.component.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+:host {
+  display: block;
+}
+
+.list {
+  transform: rotate(180deg);
+  writing-mode: vertical-lr;
+  margin-top: 32px;
+  color: $gray-2;
```
Comment (dgutoryan) [inline src/app/training/components/list-switch/list-switch.component.scss:11]: We have material theme

File: src/app/training/pages/training-page/training-page.component.html [hunk 4]
```diff
@@ -1,4 +1,14 @@
-<rs-phrase-reviewer [phrases]="phrases$ | async"></rs-phrase-reviewer>
+<div class="sidenav">
```
Comment (dgutoryan) [inline src/app/training/pages/training-page/training-page.component.html:1]: Use angular material sidebar

File: src/app/training/components/list-switch/list-switch.component.scss (hunk 3 above)
Comment (dbl7) [reply src/app/training/components/list-switch/list-switch.component.scss:11]: good idea!

**Pull request merged**